internal/human/marshal.go

```diff
@@ -237,10 +237,24 @@ func marshalSlice(slice reflect.Value, opt *MarshalOpt) (string, error) {
 		item := slice.Index(i)
 		row := []string(nil)
 		for _, fieldSpec := range opt.Fields {
-			value := getFieldValue(item, fieldSpec.FieldName)
+			fieldValue := getFieldValue(item, fieldSpec.FieldName)
 			str := ""
-			if value.IsValid() {
-				str, _ = Marshal(value.Interface(), opt)
+			if fieldValue.IsValid() {
+				var err error
```

Comment:
why is this necessary ? 
can't we use `:=` notation below ?

Reply:
Nop, `str` is already defined. https://play.golang.com/p/NwyLo8zmcMm